src/resources/packages/classy/types/TicketComponentProps.d.ts

```diff
@@ -0,0 +1,5 @@
+export type TicketComponentProps = {
```

Comment:
If this type is used only in one place (this is what I got from searching the PR); then it could be inlined in the component using it, or in a `Components.d.ts` or `Ticket.d.ts` file.

Reply:
It is used across multiple components. But as I work this feature I'll evaluate whether to keep it or inline it.